README.md

```diff
@@ -669,6 +669,105 @@ you've fixed the error, it's usually safe to re-run the `spin-up` or `tear-down`
 for the modules that already deployed successfully, and should only affect the ones that had an error the last time
 around.
 
+## Remote Terraform configurations
+
+Terragrunt has the ability to work with Terraform configurations that are not in your working directory. This is 
+especially useful if you have versioned all of your Terraform code and want to `apply` a specific version of it
+(see [Using Pipelines to Manage Environments with Infrastructure as 
+Code](https://medium.com/@kief/https-medium-com-kief-using-pipelines-to-manage-environments-with-infrastructure-as-code-b37285a1cbf5)).
+
```

Comment:
Good point. I just updated the README to make the problem clearer: https://github.com/gruntwork-io/terragrunt/pull/111/commits/cd3f23b497d5ba903a774ec4bd8fb5dae0ef2a77